Feat(OnboardingRegistration): Creates registration page

vakodrazan opened the pull request:
Hello Team,

The registration functionalities are ready for review. Please have a look at it whenever you are free. Feel free to leave some comments on things that need to improve.

Thanks in advance,

[Task link](https://app.clickup.com/t/41k8d6)
Refs: 41k8d6

andrzej-onja commented on src/App.tsx:
```diff
@@ -1,9 +1,10 @@
 import './App.css'
 import { HomePage } from './Pages/HomePage'
-import { RoleSelectOnboard } from './components/RoleSelectionOnboard/RoleSelectionOnBoard'
+// import { RoleSelectOnboard } from './components/RoleSelectionOnboard/RoleSelectionOnBoard'
```
remove unused

andrzej-onja commented on src/Pages/OnboardingPage.tsx:
```diff
@@ -0,0 +1,57 @@
+const RegisteredSuccessFully = () => (
+  <div>
+    <h1>Registered Succesfully</h1>
+  </div>
```
This doesn't match the design

andrzej-onja commented on src/components/RoleSelectionOnboard/RoleSelectionOnBoard.tsx:
```diff
@@ -17,64 +11,47 @@ import userIconHovered from '../../stories/assets/userNotHovered.svg'
+export const RoleSelectOnboard: FC<HandlerProp> = ({ setRoleSelected }) => {
   /* eslint-disable @typescript-eslint/no-unused-vars */
-  const [roleSelected, setRoleSelected] = useState('')
+  // const [roleSelected, setRoleSelected] = useState('')
```
remove if unused

andrzej-onja commented on src/components/RoleSelectionOnboard/RoleSelectionOnBoardiStyle.tsx:
```diff
@@ -16,9 +16,10 @@ export const TopContainerStyles = styled.div`
+  /* position: absolute;
   transform: translateX(-50%);
-  left: 50%;
+  left: 50%; */
```
remove commented out code

vakodrazan committed:
Feat(OnboardingRegistration): Creates registration page

Adds redux user slice

Updates grapgql helper to register user

Creates onboarding page where to select role and direct to the registration form

Updates store

Refs: 41k8d6

andrzej-onja commented on src/Pages/LoginPage.tsx:
```diff
@@ -74,65 +69,60 @@ export const LoginPage: React.FC<LoginPageProps> = ({
+                />
+              </RegistrationButtonContainer>
+            </RegistrationForm>
+          </RegistrationOnboardingContent>
```
Why registration is in Login page ?

andrzej-onja replied:
It should be renamed to login. I see its not part of your PR so live it as it is just add todo at the top that it needs to be renamed please. Approving in advance
